FreeRTOS/FreeRTOS-Plus-TCP
# Add Nxp1060 network interface

Description
-----------
Add a FreeRTOS+TCP compatible network interface for this board: https://github.com/FreeRTOS/iot-reference-nxp-rt1060

Test Steps
-----------


Checklist:
----------



Related Issue
-----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Log
------
[DHCP_Check_With_Cable_Disconnects](https://user-images.githubusercontent.com/60444055/225434073-61802d0d-d93b-411f-a638-8844216327e6.jpeg)
